src/components/annotate/task-detail/page-client.tsx

```diff
@@ -194,7 +196,36 @@ export function AnnotationTaskDetailPageClient({
                 disabled={lockingOutFromSpecies || lockingOutFromSex}
               />
             </div>
+            <div className="space-y-2">
+              <h3 className="text-foreground text-sm font-bold">Notes</h3>
+              <Textarea
+                className="h-[70px] resize-none text-sm overflow-y-auto"
+                placeholder="Add observations..."
+              />
+            </div>
+
+            <div className = "flex gap-3 pt-2">
+              <Button 
```

Comment:
[Suggestion]: I think this should be a toggle component, because you are toggling the flagged and not flagged state. Check out the UI on the login branch. Also, currently when I click the button, I get no feedback that I have flagged the specimen. The older UI made it flash and turn red. Check out the code on the login branch. Run the code on the login branch and check out the UI yourself and play around with it.